lib/routes.js

```diff
@@ -9,14 +9,34 @@ routes.prototype.get = function (args, done) {
   var endpoint = 'routes/'
   var qs = this.client.getQS(_qsAllowedProps, args)
 
-  // require route id
-  if (typeof args.id === 'undefined') {
-    throw new Error('args must include an route id')
-  }
+  _requireRouteId(args)
 
   endpoint += args.id + '?' + qs
   return this.client.getEndpoint(endpoint, args, done)
 }
+
+routes.prototype.getFile = function (args, done) {
+  var endpoint = 'routes/'
+
+  _requireRouteId(args)
+
+  this._getFileHelper(endpoint, args, done)
+}
 //= ==== routes endpoint =====
 
+//= ==== helpers =====
+var _requireRouteId = function (args) {
+  if (typeof args.id === 'undefined') {
```

Comment:
I believe the ID is always a number. Please strengthen the argument check here to check that it's a defined and a number.

Reply:
It is actually a long. To test I had to pass it as a string.
https://developers.strava.com/docs/reference/#api-Routes-getRouteAsGPX